src/main/java/com/rk/rentacarapp/controller/ViewController.java

```diff
@@ -0,0 +1,36 @@
+package com.rk.rentacarapp.controller;
+
+import com.rk.rentacarapp.model.Vehicle;
+import com.rk.rentacarapp.model.VehicleType;
+import com.rk.rentacarapp.services.VehicleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "api/view", produces = MediaType.APPLICATION_JSON_VALUE)
+public class ViewController {
+
+    @Autowired
+    private VehicleService vehicleService;
+
+    @GetMapping("/vehicles/vehicleTypes")
```

Comment:
Hm this is a ViewController and has all the endpoints towards vehicles. Why is it separated from the VehicleController?